lib/routes/admin-api/feature-schema.js

```diff
@@ -10,6 +10,22 @@ const strategiesSchema = joi.object().keys({
     parameters: joi.object(),
 });
 
+const variantsSchema = joi.object().keys({
+    name: nameType,
```

Comment:
Todo: Name cannot be 'disabled', this will produce unexpected behaviour as the fallback variant will use that name.

Reply:
Not really a problem.